services/bugmon-win/launch.sh

```diff
@@ -28,8 +28,8 @@ EOF
 
 retry python -m pip install git+https://github.com/MozillaSecurity/bugmon-tc.git
 
-ARTIFACT_DEST="$USERPROFILE/bugmon-artifacts"
-TC_ARTIFACT_ROOT="project/fuzzing/bugmon"
+ARTIFACT_DEST="./bugmon-artifacts"
```

Comment:
You could use `$PWD` if you do want it to be absolute.

Reply:
Done.